[CI] ExponentiallyWeightedMovingRateTests testEwmr_threadSafe failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #6993 / almalinux-9_platform-support-unix](https://gradle-enterprise.elastic.co/s/fz7wro3vcla3m)
- [elasticsearch-pull-request #62317 / part-1](https://gradle-enterprise.elastic.co/s/5fdkndahorvn6)
- [elasticsearch-periodic-platform-support #6986 / almalinux-8-aarch64_checkpart1_platform-support-arm](https://gradle-enterprise.elastic.co/s/ywgog4iphm4cq)
- [elasticsearch-periodic-platform-support #6966 / ubuntu-2004-aarch64_checkpart1_platform-support-arm](https://gradle-enterprise.elastic.co/s/ible4epcbhrwg)

**Reproduction Line:**
```
./gradlew ":server:test" --tests "org.elasticsearch.common.metrics.ExponentiallyWeightedMovingRateTests.testEwmr_threadSafe" -Dtests.seed=A1D9D6C0CD0FC7B2 -Dtests.locale=en-VU -Dtests.timezone=US/Pacific -Druntime.java=23
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.common.metrics.ExponentiallyWeightedMovingRateTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testEwmr_threadSafe),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: a numeric value within <1.0E-13> of <5.056759216300853>
     but: <5.056759216300751> differed by <2.1405182655143987E-15> more than delta <1.0E-13>
```

**Issue Reasons:**
- [main] 4 failures in test testEwmr_threadSafe (1.7% fail rate in 235 executions)
- [main] 3 failures in pipeline elasticsearch-periodic-platform-support (60.0% fail rate in 5 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -311,9 +311,6 @@ tests:
 - class: org.elasticsearch.multiproject.test.CoreWithMultipleProjectsClientYamlTestSuiteIT
   method: test {yaml=search.vectors/41_knn_search_byte_quantized/kNN search plus query}
   issue: https://github.com/elastic/elasticsearch/issues/124687
-- class: org.elasticsearch.common.metrics.ExponentiallyWeightedMovingRateTests
-  method: testEwmr_threadSafe
-  issue: https://github.com/elastic/elasticsearch/issues/124692
 - class: org.elasticsearch.packaging.test.BootstrapCheckTests
   method: test20RunWithBootstrapChecks
   issue: https://github.com/elastic/elasticsearch/issues/124940

--- a/server/src/test/java/org/elasticsearch/common/metrics/ExponentiallyWeightedMovingRateTests.java
+++ b/server/src/test/java/org/elasticsearch/common/metrics/ExponentiallyWeightedMovingRateTests.java
@@ -23,7 +23,7 @@ import static org.hamcrest.Matchers.equalTo;
 
 public class ExponentiallyWeightedMovingRateTests extends ESTestCase {
 
-    private static final double TOLERANCE = 1.0e-13;
+    private static final double TOLERANCE = 2.0e-13;
     private static final double HALF_LIFE_MILLIS = 1.0e6; // Half-life of used by many tests
     private static final double LAMBDA = Math.log(2.0) / HALF_LIFE_MILLIS; // Equivalent value of lambda
     public static final int START_TIME_IN_MILLIS = 1234567;

